web/styles/left_sidebar.css

```diff
@@ -372,6 +390,10 @@
         .stream-list-section-toggle {
             opacity: var(--opacity-sidebar-heading-hover);
         }
+
+        .add_stream_icon {
+            display: grid;
```

Comment:
This isn't accurate...there's no grid defined here. Would it be okay to just use `visibility` on `.add_stream_icon`? I'd think so.

Reply:
Yup that works, changed to that thanks :)